### PR TITLE
Removed restrictions and fixed other minor issues.

### PR DESCRIPTION
**Pull request by kreinold**
[DM-505](https://broadworkbench.atlassian.net/browse/DM-505)

### CODE REVIEW

File: src/terra-core/TerraCoreDataModel.ttl [hunk 1]
```diff
@@ -1694,11 +1644,11 @@ TerraCore:hasAnatomicalRegion
   rdfs:domain TerraCore:SingleCell ;
   rdfs:label "hasAnatomicalRegion" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:anyURI 
-      xsd:string
-    );
-  ];
+      owl:unionOf (
+          xsd:anyURI
```
[the remark this answers is not in the document]
Comment (rsc3) [reply src/terra-core/TerraCoreDataModel.ttl:1648]: Repeated several times below

File: src/terra-core/TerraCoreDataModel.ttl [hunk 2]
```diff
@@ -2296,11 +2242,11 @@ TerraCore:hasHostOrganism
   rdfs:domain TerraCore:Antibody ;
   rdfs:label "has host organism" ;
   rdfs:range [
-    owl:unionOf (
-      obo:OBI_0100026 
-      xsd:string 
-    );
-  ];
+      owl:unionOf (
+          obo:OBI_0100026
```
Comment (rsc3) [inline src/terra-core/TerraCoreDataModel.ttl:2246]: Can't unionOf object and datatype

File: src/terra-core/TerraCoreDataModel.ttl [hunk 3]
```diff
@@ -2791,11 +2737,11 @@ TerraCore:hasSequenceLocation
   rdfs:domain TerraCore:VariantCall ;
   rdfs:label "hasSequenceLocation" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:string 
-      TerraCore:SequenceLocation 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
Comment (rsc3) [inline src/terra-core/TerraCoreDataModel.ttl:2741]: Can't unionOf object with datatype

File: src/terra-core/TerraCoreDataModel.ttl [hunk 4]
```diff
@@ -3112,11 +3058,11 @@ TerraCore:usesAntibody
   rdfs:domain TerraCore:AssayActivity ;
   rdfs:label "usesAntibody" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:string
-      TerraCore:Antibody 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
Comment (rsc3) [inline src/terra-core/TerraCoreDataModel.ttl:3062]: can't unionOf object with datatype

File: src/terra-core/TerraCoreDataModel.ttl [hunk 5]
```diff
@@ -3142,13 +3088,13 @@ TerraCore:usesReferenceAssembly
...
-    rdfs:range [
-    owl:unionOf (
-      xsd:anyURI 
-      xsd:string
-      TerraCore:ReferenceAssembly
-    );
-  ];
+  rdfs:range [
+      owl:unionOf (
+          xsd:anyURI
```
Comment (rsc3) [inline src/terra-core/TerraCoreDataModel.ttl:3093]: can't unionOf datatypes and object

File: src/terra-core/TerraCoreDataModel.ttl [hunk 6]
```diff
@@ -3260,11 +3181,11 @@ TerraDCAT_ap:hasOwner
 .
 TerraDCAT_ap:hasPrincipalInvestigator
   rdfs:range [
-    owl:unionOf (
-      xsd:string
-      TerraCore:PrincipalInvestigator 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
Comment (rsc3) [inline src/terra-core/TerraCoreDataModel.ttl:3185]: can't unionOf datatype and object

**Review by rsc3 — CHANGES_REQUESTED** [review]

File: src/terra-core/TerraCoreDataModel.ttl (hunk 1 above)
[the remark this answers is not in the document]
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:1648]: The reason for [anyURI ](http://books.xmlschemata.org/relaxng/ch19-77009.html) is that this data type imposes restrictions on xsd:string which allow some validation to be performed if we know we have a URI.  See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl [hunk 7]
```diff
@@ -1559,11 +1509,11 @@ TerraCore:confirmsDisease
   rdfs:domain TerraCore:Diagnosis ;
   rdfs:label "confirmsDisease" ;
   rdfs:range [
-    owl:unionOf (
-      obo:OGMS_0000031 
-      xsd:anyURI
-    ) ;
-  ];
+      owl:unionOf (
```
[the remark this answers is not in the document]
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:1512]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl (hunk 6 above)
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:3185]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl (hunk 4 above)
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:3062]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl [hunk 8]
```diff
@@ -2815,11 +2761,11 @@ TerraCore:hasServiceProvider
   rdfs:domain TerraCore:Activity ;
   rdfs:label "has service provider" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:string
-      prov:Organization 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```
[the remark this answers is not in the document]
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:2765]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl (hunk 3 above)
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:2741]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl (hunk 2 above)
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:2246]: Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)

File: src/terra-core/TerraCoreDataModel.ttl [hunk 9]
```diff
@@ -1830,13 +1780,13 @@ TerraCore:hasChild
...
-      TerraCore:Donor 
-      TerraCore:FamilyMember 
-      TerraCore:HumanDonor 
-    );
-  ];
+  rdfs:range [
+      owl:unionOf (
+          TerraCore:Donor
+          TerraCore:FamilyMember
+          TerraCore:HumanDonor
```
[the remark this answers is not in the document]
Comment (kreinold) [reply src/terra-core/TerraCoreDataModel.ttl:1787]: Fix in additional checkin.

**Comment by kreinold** [comment]
PR has been updated with the following:
•	Removed foaf equivalent classes; no longer used. 
•	Fixed typo in definition
•	hasAssayType should be an Object Property
•	Removed HumanDonor from range where Donor was already specified. (per comment)

**Review by rsc3 — CHANGES_REQUESTED** [review]

**Pull request merged**